<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Publications</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/united.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>




<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">EvoGen</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    The lab
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    <span class="fa fa-flask"></span>
     
    Research
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="research.html">Topics</a>
    </li>
    <li>
      <a href="collaborators.html">Collaborators</a>
    </li>
  </ul>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-book"></span>
     
    Publications
  </a>
</li>
<li>
  <a href="jc.html">
    <span class="fa fa-newspaper"></span>
     
    Journal club
  </a>
</li>
<li>
  <a href="open.html">
    <span class="fa fa-graduation-cap"></span>
     
    Join US
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    <span class="fa fa-users"></span>
     
    Team
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-header">PI</li>
    <li>
      <a href="gossmann.html">Gossmann</a>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">Post-docs</li>
    <li>
      <a href="quentin.html">Quentin Foucault</a>
    </li>
    <li>
      <a href="vendrami.html">David Vendrami</a>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">PhDs</li>
    <li>
      <a href="yuchi.html">Yu-Chi Chen</a>
    </li>
    <li class="divider"></li>
    <li>
      <a href="students.html">Students</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:&lt;toni.gossmann@gmail.com&gt;">
    <span class="fa fa-at"></span>
     
  </a>
</li>
<li>
  <a href="contact.html">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<p><strong>All publications</strong></p>
<p>If you cannot find a copy of the paper you are looking for, please
send an Email (upper right corner).</p>
<div id="refs" class="references csl-bib-body hanging-indent"
line-spacing="2">
<div id="ref-Gossmann2022" class="csl-entry">
1. Gossmann, T. I., &amp; Waxman, D. (2022). Correcting bias in allele
frequency estimates due to an observation threshold: A markov chain
analysis. <em>Genome Biology and Evolution</em>, <em>14</em>(4). <a
href="https://doi.org/10.1093/gbe/evac047">https://doi.org/10.1093/gbe/evac047</a>
</div>
<div id="ref-Mavreas2022" class="csl-entry">
2. Mavreas, K., Gossmann, T. I., &amp; Waxman, D. (2022). Loss and
fixation of strongly favoured new variants: Understanding and extending
haldane’s result via the wright<span></span>fisher model.
<em>Biosystems</em>, <em>221</em>, 104759. <a
href="https://doi.org/10.1016/j.biosystems.2022.104759">https://doi.org/10.1016/j.biosystems.2022.104759</a>
</div>
<div id="ref-Muenzner2022" class="csl-entry">
3. Muenzner, J., Trébulle, P., Agostini, F., Messner, C. B., Steger, M.,
Lehmann, A., Caudal, E., Egger, A.-S., Amari, F., Barthel, N., Chiara,
M. D., Mülleder, M., Demichev, V., Liti, G., Schacherer, J., Gossmann,
T., Berman, J., &amp; Ralser, M. (2022). <em>The natural diversity of
the yeast proteome reveals chromosome-wide dosage compensation in
aneuploids</em>. <a
href="https://doi.org/10.1101/2022.04.06.487392">https://doi.org/10.1101/2022.04.06.487392</a>
</div>
<div id="ref-Vendrami2022" class="csl-entry">
4. Vendrami, D. L. J., Gossmann, T. I., Chakarov, N., Paijmans, A. J.,
Litzke, V., Eyre-Walker, A., Forcada, J., &amp; Hoffman, J. I. (2022).
Signatures of selection on mitonuclear integrated genes uncover hidden
mitogenomic variation in fur seals. <em>Genome Biology and
Evolution</em>, <em>14</em>(7). <a
href="https://doi.org/10.1093/gbe/evac104">https://doi.org/10.1093/gbe/evac104</a>
</div>
<div id="ref-Hildebrand2021" class="csl-entry">
5. Hildebrand, F., Gossmann, T. I., Frioux, C., Özkurt, E., Myers, P.
N., Ferretti, P., Kuhn, M., Bahram, M., Nielsen, H. B., &amp; Bork, P.
(2021). Dispersal strategies shape persistence and evolution of human
gut bacteria. <em>Cell Host <span>&amp;</span> Microbe</em>. <a
href="https://doi.org/10.1016/j.chom.2021.05.008">https://doi.org/10.1016/j.chom.2021.05.008</a>
</div>
<div id="ref-Junker2021" class="csl-entry">
6. Junker, N., &amp; Gossmann, T. I. (2021). Adaptation-driven evolution
of sirtuin 1 (<span>SIRT</span>1), a key regulator of metabolism and
aging, in marmot species. <em>Frontiers in Ecology and Evolution</em>,
<em>9</em>. <a
href="https://doi.org/10.3389/fevo.2021.666564">https://doi.org/10.3389/fevo.2021.666564</a>
</div>
<div id="ref-Stroemland2021" class="csl-entry">
7. Strømland, Øyvind, Kallio, J. P., Pschibul, A., Skoge, R. H.,
Harardóttir, H. M., Sverkeli, L. J., Heinekamp, T., Kniemeyer, O.,
Migaud, M., Makarov, M. V., Toni I. Gossmann, Brakhage, A. A., &amp;
Ziegler, M. (2021). Discovery of fungal surface <span>NADases</span>
predominantly present in pathogenic species. <em>Nature
Communications</em>, <em>12</em>(1). <a
href="https://doi.org/10.1038/s41467-021-21307-z">https://doi.org/10.1038/s41467-021-21307-z</a>
</div>
<div id="ref-Waterman2021" class="csl-entry">
8. Waterman, J. M., Gossmann, T. I., Brandler, O., &amp; Koprowski, J.
L. (2021). <em>Editorial: Ecological, behavioral and genomic
consequences in the rodent family sciuridae: Why are squirrels so
diverse?</em> <em>9</em>. <a
href="https://doi.org/10.3389/fevo.2021.765558">https://doi.org/10.3389/fevo.2021.765558</a>
</div>
<div id="ref-Yusuf_2020" class="csl-entry">
9. Yusuf, L., Heatley, M. C., Palmer, J. P. G., Barton, H. J., Cooney,
C. R., &amp; Toni I. Gossmann. (2020). Noncoding regions underpin avian
bill shape diversification at macroevolutionary scales. <em>Genome
Research</em>. <a
href="https://doi.org/10.1101/gr.255752.119">https://doi.org/10.1101/gr.255752.119</a>
</div>
<div id="ref-Gossmann2020" class="csl-entry">
10. Toni I. Gossmann, &amp; Ralser, M. (2020). Marmota marmota.
<em>Trends in Genetics</em>. <a
href="https://doi.org/10.1016/j.tig.2020.01.006">https://doi.org/10.1016/j.tig.2020.01.006</a>
</div>
<div id="ref-Bockwoldt2019" class="csl-entry">
11. Bockwoldt, M., Houry, D., Niere, M., Gossmann, Toni I., Reinartz,
I., Schug, A., Ziegler, M., &amp; Heiland, I. (2019). Identification of
evolutionary and kinetic drivers of NAD-dependent signaling.
<em>Proceedings of the National Academy of Sciences of the United States
of America</em>. <a
href="https://doi.org/10.1073/pnas.1902346116">https://doi.org/10.1073/pnas.1902346116</a>
</div>
<div id="ref-Hildebrand2019" class="csl-entry">
12. Hildebrand, F., Moitinho-Silva, L., Blasche, S., Jahn, M. T., Toni I
Gossmann, Huerta-Cepas, J., Hercog, R., Luetge, M., Bahram, M.,
Pryszlak, A., Alves, R. J., Waszak, S. M., Zhu, A., Ye, L., Costea, P.
I., Aalvink, S., Belzer, C., Forslund, S. K., Sunagawa, S., … Bork, P.
(2019). <span>A</span>ntibiotics-induced monodominance of a novel gut
bacterial order. <em>Gut</em>, gutjnl-2018-317715. <a
href="https://doi.org/10.1136/gutjnl-2018-317715">https://doi.org/10.1136/gutjnl-2018-317715</a>
</div>
<div id="ref-Karr2019" class="csl-entry">
13. Karr, T. L., Southern, H., Rosenow, M. A., Toni I. Gossmann, &amp;
Snook, R. R. (2019). <span>T</span>he old and the new: Discovery
proteomics identifies putative novel seminal fluid proteins in
<span>D</span>rosophila. <em>Molecular <span>&amp;</span> Cellular
Proteomics</em>, mcp.RA118.001098. <a
href="https://doi.org/10.1074/mcp.ra118.001098">https://doi.org/10.1074/mcp.ra118.001098</a>
</div>
<div id="ref-Kinsella2019" class="csl-entry">
14. Kinsella, C. M., Ruiz-Ruano, F. J., Dion-Côté, A.-M., Charles, A.
J., Gossmann, T. I., Cabrero, J., Kappei, D., Hemmings, N., Simons, M.
J. P., Camacho, J. P. M., Forstmeier, W., &amp; Suh, A. (2019).
Programmed <span>DNA</span> elimination of germline development genes in
songbirds. <em>Nature Communications</em>, <em>10</em>(1). <a
href="https://doi.org/10.1038/s41467-019-13427-4">https://doi.org/10.1038/s41467-019-13427-4</a>
</div>
<div id="ref-Laine2019" class="csl-entry">
15. Laine, V. N., Gossmann, Toni I, Oers, K. van, Visser, M. E., &amp;
Groenen, M. A. M. (2019). <span>E</span>xploring the unmapped
<span>DNA</span> and <span>RNA</span> reads in a songbird genome.
<em>BMC Genomics</em>, <em>20</em>, 19. <a
href="https://doi.org/10.1186/s12864-018-5378-2">https://doi.org/10.1186/s12864-018-5378-2</a>
</div>
<div id="ref-Gossmann2019" class="csl-entry">
16. Toni I. Gossmann, Shanmugasundram, A., Börno, S., Duvaux, L.,
Lemaire, C., Kuhl, H., Klages, S., Roberts, L. D., Schade, S., Gostner,
J. M., Hildebrand, F., Vowinckel, J., Bichet, C., Mülleder, M., Calvani,
E., Zelezniak, A., Griffin, J. L., Bork, P., Allaine, D., … Ralser, M.
(2019). <span>I</span>ce-<span>A</span>ge <span>C</span>limate
<span>A</span>daptations <span>T</span>rap the <span>A</span>lpine
<span>M</span>armot in a <span>S</span>tate of <span>L</span>ow
<span>G</span>enetic <span>D</span>iversity. <em>Current Biology</em>.
<a
href="https://doi.org/10.1016/j.cub.2019.04.020">https://doi.org/10.1016/j.cub.2019.04.020</a>
</div>
<div id="ref-Afanasyeva2018" class="csl-entry">
17. Afanasyeva, A., Bockwoldt, M., Cooney, C. R., Heiland, I., &amp;
Gossmann, Toni I. (2018). <span>H</span>uman long intrinsically
disordered protein regions are frequent targets of positive selection.
<em>Genome Research</em>, <em>28</em>, 975–982. <a
href="https://doi.org/10.1101/gr.232645.117">https://doi.org/10.1101/gr.232645.117</a>
</div>
<div id="ref-Gossmann2018b" class="csl-entry">
18. Toni I. Gossmann, Bockwoldt, M., Diringer, L., Schwarz, F., &amp;
Schumann, V.-F. (2018). <span>E</span>vidence for <span>S</span>trong
<span>F</span>ixation <span>B</span>ias at 4-fold
<span>D</span>egenerate <span>S</span>ites <span>A</span>cross
<span>G</span>enes in the <span>G</span>reat <span>T</span>it
<span>G</span>enome. <em>Frontiers in Ecology and Evolution</em>,
<em>6</em>. <a
href="https://doi.org/10.3389/fevo.2018.00203">https://doi.org/10.3389/fevo.2018.00203</a>
</div>
<div id="ref-Corcoran2017" class="csl-entry">
19. Corcoran, P., Gossmann, Toni I, Barton, H. J., Slate, J., &amp;
Zeng, K. (2017). <span>D</span>eterminants of the efficacy of natural
selection on coding and noncoding variability in two passerine species.
<em>Genome Biology and Evolution</em>.
</div>
<div id="ref-Kappei2017" class="csl-entry">
20. Kappei, D., Scheibe, M., Paszkowski-Rogacz, M., Bluhm, A., Gossmann,
Toni I., Dietz, S., Dejung, M., Herlyn, H., Buchholz, F., Mann, M.,
&amp; Butter, F. (2017). <span>P</span>hylointeractomics reconstructs
functional evolution of protein binding. <em>Nature Communications</em>,
<em>8</em>, 14334. <a
href="https://doi.org/10.1038/ncomms14334">https://doi.org/10.1038/ncomms14334</a>
</div>
<div id="ref-Shafiey2017" class="csl-entry">
21. Shafiey, H., Gossmann, Toni I, &amp; Waxman, D. (2017).
<span>E</span>volutionary control: <span>T</span>argeted change of
allele frequencies in natural populations using externally directed
evolution. <em>Journal of Theoretical Biology</em>, <em>419</em>,
362–374. <a
href="https://doi.org/10.1016/j.jtbi.2017.01.023">https://doi.org/10.1016/j.jtbi.2017.01.023</a>
</div>
<div id="ref-Laine2016" class="csl-entry">
22. Laine, V. N., Gossmann, Toni I, Schachtschneider, K. M., Garroway,
C. J., Madsen, O., Verhoeven, K. J. F., Jager, V. de, Megens, H.-J.,
Warren, W. C., Minx, P., Crooijmans, R. P. M. A., Corcoran, P.,
Consortium, G. T. H., Sheldon, B. C., Slate, J., Zeng, K., Oers, K. van,
Visser, M. E., &amp; Groenen, M. A. M. (2016). <span>E</span>volutionary
signals of selection on cognition from the great tit genome and
methylome. <em>Nature Communications</em>, <em>7</em>, 10474. <a
href="https://doi.org/10.1038/ncomms10474">https://doi.org/10.1038/ncomms10474</a>
</div>
<div id="ref-Zhao2016" class="csl-entry">
23. Zhao, L., Toni I Gossmann, &amp; Waxman, D. (2016). <span>A</span>
modified <span>W</span>right-<span>F</span>isher model that incorporates
<span>N</span>e: <span>A</span> variant of the standard model with
increased biological realism and reduced computational complexity.
<em>Journal of Theoretical Biology</em>, <em>393</em>, 218–228. <a
href="https://doi.org/10.1016/j.jtbi.2016.01.002">https://doi.org/10.1016/j.jtbi.2016.01.002</a>
</div>
<div id="ref-Gossmann2016" class="csl-entry">
24. Gossmann, Toni I., Saleh, D., Schmid, M. W., Spence, M. A., &amp;
Schmid, K. J. (2016). Transcriptomes of plant gametophytes have a higher
proportion of rapidly evolving and young genes than sporophytes.
<em>Molecular Biology and Evolution</em>, <em>33</em>, 1669–1678. <a
href="https://doi.org/10.1093/molbev/msw044">https://doi.org/10.1093/molbev/msw044</a>
</div>
<div id="ref-Gossmann2014a" class="csl-entry">
25. Gossmann, Toni I, Santure, A. W., Sheldon, B. C., Slate, J., &amp;
Zeng, K. (2014). <span>H</span>ighly variable recombinational landscape
modulates efficacy of natural selection in birds. <em>Genome Biology and
Evolution</em>, <em>6</em>, 2061–2075. <a
href="https://doi.org/10.1093/gbe/evu157">https://doi.org/10.1093/gbe/evu157</a>
</div>
<div id="ref-Gossmann2014c" class="csl-entry">
26. Gossmann, Toni I, Schmid, M. W., Grossniklaus, U., &amp; Schmid, K.
J. (2014). <span>S</span>election-driven evolution of sex-biased genes
is consistent with sexual selection in <span>A</span>rabidopsis
thaliana. <em>Molecular Biology and Evolution</em>, <em>31</em>,
574–583. <a
href="https://doi.org/10.1093/molbev/mst226">https://doi.org/10.1093/molbev/mst226</a>
</div>
<div id="ref-Gossmann2014b" class="csl-entry">
27. Gossmann, Toni I, Waxman, D., &amp; Eyre-Walker, A. (2014).
<span>F</span>luctuating selection models and
<span>M</span>c<span>D</span>onald-<span>K</span>reitman type analyses.
<em>PLoS One</em>, <em>9</em>, e84540. <a
href="https://doi.org/10.1371/journal.pone.0084540">https://doi.org/10.1371/journal.pone.0084540</a>
</div>
<div id="ref-Gossmann2014" class="csl-entry">
28. Gossmann, Toni I, &amp; Ziegler, M. (2014). <span>S</span>equence
divergence and diversity suggests ongoing functional diversification of
vertebrate <span>NAD</span> metabolism. <em>DNA Repair</em>,
<em>23</em>, 39–48. <a
href="https://doi.org/10.1016/j.dnarep.2014.07.005">https://doi.org/10.1016/j.dnarep.2014.07.005</a>
</div>
<div id="ref-Gossmann2012" class="csl-entry">
29. Gossmann, Toni I, Keightley, P. D., &amp; Eyre-Walker, A. (2012).
<span>T</span>he effect of variation in the effective population size on
the rate of adaptive molecular evolution in eukaryotes. <em>Genome
Biology and Evolution</em>, <em>4</em>, 658–667. <a
href="https://doi.org/10.1093/gbe/evs027">https://doi.org/10.1093/gbe/evs027</a>
</div>
<div id="ref-gossmann2012two" class="csl-entry">
30. Gossmann, Toni I, Ziegler, M., Puntervoll, P., de Figueiredo, L. F.,
Schuster, S., &amp; Heiland, I. (2012). <span>T</span>wo new
perspectives on <span>NAD</span> metabolism. <em>GCB 2012 Highlight
Abstracts</em>, 5.
</div>
<div id="ref-Gossmann2012a" class="csl-entry">
31. Gossmann, Toni I, Ziegler, M., Puntervoll, P., de Figueiredo, L. F.,
Schuster, S., &amp; Heiland, I. (2012). <span>NAD</span>(+) biosynthesis
and salvage–a phylogenetic perspective. <em>FEBS Journal</em>,
<em>279</em>, 3355–3363. <a
href="https://doi.org/10.1111/j.1742-4658.2012.08559.x">https://doi.org/10.1111/j.1742-4658.2012.08559.x</a>
</div>
<div id="ref-deFigueiredo2011" class="csl-entry">
32. de Figueiredo, L. F., Gossmann, Toni I, Ziegler, M., &amp; Schuster,
S. (2011). <span>P</span>athway analysis of <span>NAD</span>+
metabolism. <em>Biochemical Journal</em>, <em>439</em>, 341–348. <a
href="https://doi.org/10.1042/BJ20110320">https://doi.org/10.1042/BJ20110320</a>
</div>
<div id="ref-Gossmann2011" class="csl-entry">
33. Gossmann, Toni I, &amp; Schmid, K. J. (2011).
<span>S</span>election-driven divergence after gene duplication in
<span>A</span>rabidopsis thaliana. <em>Journal of Molecular
Evolution</em>, <em>73</em>, 153–165. <a
href="https://doi.org/10.1007/s00239-011-9463-2">https://doi.org/10.1007/s00239-011-9463-2</a>
</div>
<div id="ref-Gossmann2011a" class="csl-entry">
34. Gossmann, Toni I, Woolfit, M., &amp; Eyre-Walker, A. (2011).
<span>Q</span>uantifying the variation in the effective population size
within a genome. <em>Genetics</em>, <em>189</em>, 1389–1402. <a
href="https://doi.org/10.1534/genetics.111.132654">https://doi.org/10.1534/genetics.111.132654</a>
</div>
<div id="ref-Lau2010" class="csl-entry">
35. Lau, C., Dölle, C., Gossmann, Toni I, Agledal, L., Niere, M., &amp;
Ziegler, M. (2010). <span>I</span>soform-specific targeting and
interaction domains in human nicotinamide mononucleotide
adenylyltransferases. <em>Journal of Biological Chemistry</em>,
<em>285</em>, 18868–18876. <a
href="https://doi.org/10.1074/jbc.M110.107631">https://doi.org/10.1074/jbc.M110.107631</a>
</div>
<div id="ref-Gossmann2010" class="csl-entry">
36. Gossmann, Toni I, Song, B.-H., Windsor, A. J., Mitchell-Olds, T.,
Dixon, C. J., Kapralov, M. V., Filatov, D. A., &amp; Eyre-Walker, A.
(2010). <span>G</span>enome wide analyses reveal little evidence for
adaptive evolution in many plant species. <em>Molecular Biology and
Evolution</em>, <em>27</em>, 1822–1832. <a
href="https://doi.org/10.1093/molbev/msq079">https://doi.org/10.1093/molbev/msq079</a>
</div>
</div>

<p style="text-align:right">2022 EvoGen Lab (PI Gossmann).</p>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
